src/Inkluzitron/Modules/SendModule.cs

```diff
@@ -1,44 +1,70 @@
+using System.Linq;
 using System.Net.Http;
 using System.Threading.Tasks;
 using Discord;
 using Discord.Commands;
+using Inkluzitron.Models.Settings;
 
 namespace Inkluzitron.Modules
 {
     [Name("Odesílání zpráv")]
     public class SendModule : ModuleBase
     {
+        private SendSettings SendSettings { get; }
+
+        public SendModule(SendSettings sendSettings)
+        {
+            SendSettings = sendSettings;
+        }
+
         [Command("send")]
-        [Summary("Odešle anonymně zprávu (i včetně přílohy) do stejné roomky. Původní příkaz smaže.")]
-        public async Task SendAsync([Remainder][Name("zpráva")] string message = null)
+        [Summary("Odešle anonymně zprávu (včetně příloh) do zadané roomky. Funguje pouze v DM.")]
+        public async Task SendAsync([Name("název roomky")] string roomName, [Remainder][Name("zpráva")] string messageText = null)
         {
-            // get collection of message attachements
-            var attachments = Context.Message.Attachments;
+            if (!Context.IsPrivate)
```

Comment:
Pokud chceš omezit používání příkazu na DMs, tak na to je atribut `RequireContext` (`[RequireContext(ContextType.DM, ErrorMessage = "Tento příkaz funguje pouze v soukromých zprávách.")]`), který se umístí na třídu, nebo command.

Jen pozor. Pokud umístíš atribut na přímo command, tak nebude fungovat nastavení ErrorMessage. Je to nahlášený bug v Discord.NET, takže doporučuju umístit na třídu.